cyences_app_for_splunk/default/props.conf

```diff
@@ -210,12 +210,12 @@ EXTRACT-src_ip_for_radius = \sfrom \'(?<src_ip>[^\']+)\',
 
 [fgt_event]
 # fgt_event for old Fortigate Add-on (1.6.2)
-EVAL-dest_category = if(subtype="vpn" AND vendor_action IN ("negotiate", "ssl-login-fail"), "vpn_auth", dest_category)
+EVAL-dest_category = if(subtype="vpn" AND vendor_action IN ("tunnel-up", "ssl-login-fail"), "vpn_auth", dest_category)
```

Comment:
Just tunnel-up won't work, because tunnel-up also includes Network tunnels, which is not expected.